### PR TITLE
Use local assets

### LINKED ISSUE #256: avoid fetching remote resources
I just noticed that loading dimension requests 2 external resources : 

https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css (in web/public/index.html)

Is there a reason not to include it in the source tree of dimension directly ?

and  https://fonts.googleapis.com/css?family=Open+Sans:100|Roboto:300 (in web/style/app.scss)

As far as I unterstand, it downloads the font from google only if it was not found locally. Is it possible to include this font directly in the source tree, or at least the css checking for the font ?

I have no idea of web development, but I always try to avoid informing external services about every click in my infrastructure.

### PR DESCRIPTION
**Pull request by Perflyst**
Close #256

### CODE REVIEW

File: web/public/index.html [hunk 1]
```diff
@@ -82,7 +82,7 @@
         }
     </style>
 
-    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
+    <link href="assets/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
```
Comment (turt2live) [inline web/public/index.html]: ideally this would be pulled from npm rather than stored locally

File: web/style/assets/opensans100-roboto300 [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 300;
+  src: local('Roboto Light'), local('Roboto-Light'), url('opensans100-roboto300.ttf') format('truetype');
```
Comment (turt2live) [inline web/style/assets/opensans100-roboto300]: The font files should be under the `public` folder, not nested into the styles

File: web/public/index.html (hunk 1 above)
Comment (Perflyst) [reply web/public/index.html]: I have no idea on how to integrate this, sorry. I have seen the newest version is 4.3.1 here https://www.npmjs.com/package/bootstrap, worth updating too?

Comment (turt2live) [reply web/public/index.html]: usually it means using `node_modules` in a way that webpack likes

**Comment by turt2live** [comment]
Looks like there's some build failures too

File: web/public/index.html (hunk 1 above)
Comment (turt2live) [reply web/public/index.html]: I've solved this part by upgrading a bunch of dependencies. The font stuff is all that's left I think

**Pull request merged**